### PR TITLE
Project request descriptions bug

### PR DESCRIPTION
**Pull request by robbiecarlton**
Fixes this: https://trello.com/c/8T2LJFym

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity - good to merge? Tests have passed in CircleCI, think the above is just a glitch in GitHub.

File: src/components/ProjectPostEditor.js [hunk 1]
```diff
@@ -125,6 +126,14 @@ class ProjectRequestEditor extends React.Component {
     const showDetails = this.state.showDetails || !!description
     const editorClass = cx('details', {empty: !showDetails})
 
+    const onBlur = () => {
+      // we have to update description here because RichTextEditor doesn't always
+      // send onChange events
+      console.log('on blur called', this.refs.details.getContent())
```
Comment (levity) [inline src/components/ProjectPostEditor.js]: remove console.log

File: src/components/RichTextEditor.js [hunk 2]
```diff
@@ -125,8 +125,18 @@ export default class RichTextEditor extends React.PureComponent {
     } = this.props
 
     Promise.resolve(window.tinymce || loadScript(TINYMCE_ASSET_URL))
+    .then(() => new Promise(resolve => {
+      const mutex = setInterval(() => {
+        if (window.RichTextEditorMutex) return
```
Comment (levity) [inline src/components/RichTextEditor.js]: let's add a comment about what this is for, e.g. "When multiple RichTextEditor components are mounted at once, we have to serialize their initialization, or weird things happen."

**Review by levity — CHANGES_REQUESTED** [review]

**Comment by levity** [comment]
Good to merge after the two suggested changes.

**Pull request merged**